### ISSUE TITLE
Auto indent ES6 `` quoted string automaticly.

### ISSUE THREAD

**Issue by thierryc** [gandm/language-babel#314]
Small bug.
Language-babel auto indent ES6 `` quoted string automaticly. 

SRC
```jsx

export default () => (
  <div className='hello'>
    <p>Hello World 4</p>
    <style jsx>{`
      :global(:root) {
        --bgColor: green;
        --color: white;
      }
      .hello {
        font: 15px Helvetica, Arial, sans-serif;
        background: var(--bgColor);
        color: var(--color);
        padding: 100px;
        text-align: center;
        transition: 100ms ease-in background;
      }
      .hello:hover {
        color: color(var(--color) blackness(+80%));
      }
    `}</style>
  </div>
)


```

RESULT
```jsx

export default () => (
  <div className='hello'>
    <p>Hello World 4</p>
    <style jsx>{`
      :global(:root) {
      --bgColor: green;
      --color: white;
      }
      .hello {
      font: 15px Helvetica, Arial, sans-serif;
      background: var(--bgColor);
      color: var(--color);
      padding: 100px;
      text-align: center;
      transition: 100ms ease-in background;
      }
      .hello:hover {
      color: color(var(--color) blackness(+80%));
      }
    `}</style>
  </div>
)


```

**Comment by gandm** [comment]
Thanks. I've disabled autoIndentJSX indentation when embedding template literals that contain an unknown language. Published in 2.50.0

**Comment by gandm** [comment]
I should rephrase that. I disable autoIndentJSX from doing any indentation inside template literals whilst still working on surrounding JSX.
